lib/moab/storage_repository.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'find' # for object_size
```

Comment:
oddly, this isn't necessary unless you run this class's spec on its own;  maybe it is required somewhere in some dependency pulled in (?)